pkg/deps/types/appquerier.go

```diff
@@ -0,0 +1,49 @@
+package types
+
+import (
+	"context"
+
+	"cosmossdk.io/depinject"
+	"github.com/cosmos/cosmos-sdk/client"
+
+	"github.com/pokt-network/poktroll/pkg/relayer"
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
+)
+
+type ApplicationQuerier interface {
```

Comment:
same as https://github.com/pokt-network/poktroll/pull/187#discussion_r1398782145; let's move this to `pkg/client/interface.go`.